src/Tizen.NUI/src/internal/Interop/Interop.TextEditor.cs

```diff
@@ -239,6 +239,9 @@ internal static partial class TextEditor
             //manual pinvoke for text-editor ScrollStateChangedSignal
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_TextEditor_ScrollStateChangedSignal")]
             public static extern global::System.IntPtr TextEditor_ScrollStateChangedSignal(global::System.Runtime.InteropServices.HandleRef jarg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_TextEditor_Property_ENABLE_EDITING_get")]
+            public static extern int TextEditor_Property_ENABLE_EDITING_get();
```

Comment:
This is different from the api name in C # binder.
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/245082/3/dali-csharp-binder/src/text-editor.cpp

Reply:
Hi @Ali-Alzyoud, the value of `EntryPoint` should exactly same as the function name in the dali-csharp-binder.

As I see the dali-csharp-binder code, it seems that the correct one is "CSharp_TextEditor_Property_ENABLE_EDITING_get" (without Dali in the middle).

And please note that I made a mistake that I merged you previous patch @2066, but it has same problem about binding names. Please check and make new patch to fix that.

Thank you.